solanaj/src/main/java/org/p2p/solanaj/kits/renBridge/renVM/HashUtils.kt

```diff
@@ -0,0 +1,56 @@
+package org.p2p.solanaj.kits.renBridge.renVM
+
+import android.util.Log
+import org.bitcoinj.core.Base58
+import org.p2p.solanaj.kits.renBridge.renVM.types.ParamsSubmitMint
+import org.p2p.solanaj.utils.ByteUtils
+import org.p2p.solanaj.utils.Hash
+import org.p2p.solanaj.utils.Utils
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+
+object HashUtils {
+
+    fun hashTransactionMint(
+        mintTx: ParamsSubmitMint.MintTransactionInput,
+        selector: String
+    ): ByteArray {
+
+        val out = ByteArrayOutputStream()
```

Comment:
should we close the stream when we are finished?

Reply:
/**
     * Closing a <tt>ByteArrayOutputStream</tt> has no effect. The methods in
     * this class can be called after the stream has been closed without
     * generating an <tt>IOException</tt>.
     */
    public void close() throws IOException {
    }

Reply:
that's why i did not do this